.github/workflows/main.yml

```diff
@@ -24,6 +24,30 @@ jobs:
     uses: ./.github/workflows/build-lint-test.yml
     secrets: inherit
 
+  is-release:
+    name: Determine whether this is a release merge commit
+    needs: build-lint-test
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    outputs:
+      IS_RELEASE: ${{ steps.is-release.outputs.IS_RELEASE }}
+    steps:
+      - id: is-release
+        uses: MetaMask/action-is-release@dc4672b05e3b1d464cdaf783579b04a4e43f8b02
+        with:
+          commit-starts-with: 'Release [version],Release v[version],Release/[version],Release/v[version]'
```

Comment:
If you're going to convert it to a regular repo, you should use this instead: https://github.com/MetaMask/metamask-module-template/blob/076554c5ccb56be236033c31edba862f91227336/.github/workflows/main.yml#L52-L65

Reply:
done